views/js/qtiCreator/tpl/forms/response/responseForm.tpl

```diff
@@ -22,6 +22,20 @@
     </select>
 </div>
 
+{{#if textEntryInteraction}}
+<div class="panel">
+    <label for="" class="has-icon">{{__ "Response base type"}}</label>
```

Comment:
Why not set an ID to the `select`, and put a `for` value here, so that they are properly linked?

Reply:
Yes, I put here and in the rest of template :)